API/makeEmptyResultStruct.m

```diff
@@ -35,19 +35,19 @@
     % (2) result.contrastParams
 
     backgroundParams = zeros(nContrasts,1);
-    coder.varsize('backgroundParams',[10000 1],[1 0]);
+    coder.varsize('backgroundParams',[100 1],[1 0]);
```

Comment:
Where are these new values from?

Reply:
They're arbitrary, as with the previous values.

We've seen that the code uses inordinate amounts of memory so I'm proposing a limit of 100 values per data block (i.e, for each of params, background_params, scalefactors etc). This would correspond to 33 independent layers, which is far more than any of our examples use, though I accept there may be cases where this limit is too low.